platform-coop-toolkit/coop-tool-library
# feat: add aXe test to homepage

## Description

Should run axe-cli on the homepage.

## Steps to test

1. Open a PR.

**Expected behavior:** Deploy preview's homepage is tested with aXe.

## Additional information

Not applicable.

## Related issues

Not applicable.
